frappe/hooks.py

```diff
@@ -424,3 +424,22 @@
 extend_bootinfo = [
 	"frappe.utils.telemetry.add_bootinfo",
 ]
+
+get_changelog_feed = "frappe.hooks.get_feed"
+
+
+def get_feed():
```

Comment:
This should be inside some field, hooks.py usually doesn't contain any code. 


Move it to `change_log.py`